arangodb-net-standard/CollectionApi/CollectionFiguresModels/FiguresBase.cs

```diff
@@ -0,0 +1,7 @@
+﻿namespace ArangoDBNetStandard.CollectionApi
+{
+    public class FiguresBase
+    {
+        public int Count { get; set; }
```

Comment:
I don't find much value in using inheritance just because there's a common `Count` property. It might seem like a violation of DRY principle but I think it's easier not to use inheritance in this case.